hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -171,6 +168,22 @@ public long takeSnapshot() throws IOException {
     return 0;
   }
 
+  /**
+   * Leader OM has purged entries from its log. To catch up, OM must download
```

Comment:
Thanks for adding descriptive javadocs to methods. It really makes code review much easier!